src/components/LoanCards/HoverLoanCard/HoverLoanCardSmall.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+	<div class="hover-loan-card-small" :class="{expanded}">
+		<loan-card-image
+			:loan-id="loan.id"
+			:name="loan.name"
+			:retina-image-url="loan.image.retina"
+			:standard-image-url="loan.image.default"
+			:is-visitor="true"
+			@click.native="trackProfileClick"
+			:open-in-new-tab="true"
+			:use-default-styles="false"
+		/>
+		<div class="hover-loan-card-data-wrap">
+			<p class="name">
+				{{ loan.name }}
+			</p>
+			<div class="minimal-fundraising-meter">
+				<minimal-fundraising-meter
+					:amount-left="amountLeft"
+					:percent-raised="percentRaised"
+					:is-funded="loan.status==='funded'"
+				/>
+			</div>
+			<p class="small-text loan-data">
+				{{ loan.geocode.country.name }} / {{ loan.activity.name }}
+			</p>
+		</div>
+	</div>
+</template>
+<script>
+import numeral from 'numeral';
+import _get from 'lodash/get';
+import LoanCardImage from '@/components/LoanCards/LoanCardImage';
+import MinimalFundraisingMeter from '@/components/LoansYouMightLike/MinimalFundraisingMeter';
+
+export default {
+	components: {
+		LoanCardImage,
+		MinimalFundraisingMeter,
+	},
+	inject: ['apollo'],
+	props: {
+		loan: {
+			type: Object,
+			default: () => {
+				return {
+					userProperties: {},
+					loanFundraisingInfo: {},
+					geocode: {
+						country: {}
+					},
+					image: {}
+				};
+			}
+		},
+		cardNumber: {
+			type: Number,
+			default: 0
+		},
+		expanded: {
+			type: Boolean,
+			default: false,
+		},
+	},
+	computed: {
+		amountLeft() {
+			const { fundedAmount, reservedAmount } = this.loan.loanFundraisingInfo;
+			return numeral(this.loan.loanAmount).subtract(fundedAmount).subtract(reservedAmount).value();
+		},
+		isFunded() {
+			return this.loan.status === 'funded' || this.amountLeft <= 0;
+		},
+		percentRaised() {
+			return numeral(numeral(this.loan.loanAmount).subtract(this.amountLeft))
+				.divide(this.loan.loanAmount).value();
+		},
+		loanType() {
```

Comment:
This can eventually be removed now or later is fine. The tracking method that uses it will be re-factored for yet to be determined requirements...